pwa/src/templates/componentDetail/ComponentsDetailTemplate.tsx

```diff
@@ -258,6 +243,7 @@ export const ComponentsDetailTemplate: React.FC<ComponentsDetailTemplateProps> =
                   secondaryButton={{
                     label: t("Close"),
                     icon: <ArrowLeftIcon />,
+                    // eslint-disable-next-line @typescript-eslint/no-empty-function
```

Comment:
Could you ask @NovaEBA or @britt11654 to create a issue about this? We should fix the underlying `NotificationPopUp` component, so that we don't _have_ to pass the `handleClick`. When we've fixed this, we can remove this `eslint-disable`.

For now however, this is fine.